runner/run.go

```diff
@@ -26,13 +27,29 @@ import (
 func Run(include string, exclude string, showTime bool, output bool, ftwtests []test.FTWTest) TestRunContext {
 	printUnlessQuietMode(output, ":rocket:Running go-ftw!\n")
 
+	// TODO: this is a hack, there should only be one instance of FTWLogLines
+	logLines := &waflog.FTWLogLines{
+		FileName: config.FTWConfig.LogFile,
+	}
+	var logFile *os.File
+	if logLines.FileName != "" {
+		var err error
+		logFile, err = os.Open(logLines.FileName)
+		if err != nil {
+			log.Panic().Caller().Err(err).Msg("failed to open file")
+		}
+		defer logFile.Close()
+	}
```

Comment:
This can easily be moved to a singleton pattern, right?

Reply:
Yes. The only other object that uses FTWLogLines is the check object. And the only reason I need FTWLogLines is because I put the methods there (they seem to fit well in there). If you have an idea, go for it, I wasn't sure what the best way would be to proceed.